khard/cli.py

```diff
@@ -278,6 +278,12 @@ def create_parsers() -> Tuple[argparse.ArgumentParser,
     add_email_parser.add_argument(
         "--vcard-version", choices=("3.0", "4.0"), dest='preferred_version',
         help="Select preferred vcard version for new contact")
+    add_email_parser.add_argument(
+        "-H", "--header", dest='field', default="From", help="Extract contacts from the given header field"
+        )
+    add_email_parser.add_argument(
+        "--all", dest="field", action="store_const", const="all", help="Extract contacts from all headers"
+        )
```

Comment:
Sure, I'll run a formatter over it.

Perhaps we could just have the `-H` option for now with the comment in the docs. I'd find it easier adding a new argument later rather than removing one.

Yes, I'm just having a play around with the cli parsing now. I've just checked the field fetching code and it converts both to lower before comparison so we should be safe here.